.changes/15fb7dc6-dee9-4f40-adba-a48183d4f432.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "15fb7dc6-dee9-4f40-adba-a48183d4f432",
+    "type": "bugfix",
+    "description": "Paginators not respecting start key",
```

Comment:
nit: usually the changelog description should finish the sentence "This PR will..." 

"Fix paginators not respecting start key"